.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think this should be removed from the repo and add .vscode to .gitignore

Reply:
fixed